.kitchen.vagrant.yml

```diff
@@ -0,0 +1,30 @@
+---
```

Comment:
I would suggest to remove this one completely as we are not going to test anything else than docker.